src/main/java/com/frauas/exercisegenerator/documents/User.java

```diff
@@ -1,19 +1,26 @@
 package com.frauas.exercisegenerator.documents;
 
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.NoArgsConstructor;
 import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.index.Indexed;
 import org.springframework.data.mongodb.core.mapping.Document;
 
-import lombok.Builder;
 import lombok.Data;
 
 @Data
 @Builder
-@Document(collection = "users")
+@AllArgsConstructor
+@NoArgsConstructor
+@Document
 public class User {
 
     @Id
     private String id;
 
-    private String userName;
+    //@Indexed(unique = true)
```

Comment:
Das FE braucht ein unique Property im Token, um den User aus dem BE zu holen. Entweder username unique oder ID im Token mitsenden.